pkg/host/internal/sriov/sriov.go

```diff
@@ -117,6 +117,7 @@ func (s *sriov) GetVfInfo(pciAddr string, devices []*ghw.PCIDevice) sriovnetwork
 		vf.Name = name
 		vf.Mac = s.networkHelper.GetNetDevMac(name)
 	}
+	vf.GUID = s.networkHelper.GetNetDevNodeGUID(pciAddr)
```

Comment:
should we check here if the string is empty we don't allocate it?
this way it will not be showed in the nodeState

Reply:
GUID is configured with the json tag `omitempty`. https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/d8ddad1adfa06e34409cfa47ad268b1781671952/api/v1/sriovnetworknodestate_types.go#L87

For a field of type `string`, this means that if the string you pass in is empty, the field will be omitted. https://pkg.go.dev/encoding/json#:~:text=The%20%22omitempty%22%20option%20specifies%20that%20the%20field%20should%20be%20omitted%20from%20the%20encoding%20if%20the%20field%20has%20an%20empty%20value%2C%20defined%20as%20false%2C%200%2C%20a%20nil%20pointer%2C%20a%20nil%20interface%20value%2C%20and%20any%20empty%20array%2C%20slice%2C%20map%2C%20or%20string.